src/EventTeachingService.h

```diff
@@ -28,8 +28,10 @@ class EventTeachingService : public Service
 private:
   Controller *controller;
   Configuration *module_config;  // Shortcut to reduce indirection code.
+  byte (*checkProduced)();
 
   bool bLearn = false;
+  unsigned long timeOut = 5000;
```

Comment:
Need a comment to clarify what this is a timeout for.
Also, as it is never changed I would make it "const" and give it a name with all upper case letters.

Reply:
OK.